pangeo-notebook/environment.yml

```diff
@@ -3,6 +3,9 @@ channels:
  - conda-forge
 dependencies:
  - python=3.9*
+ # needed for pytorch to work
+ # https://github.com/NVIDIA/MinkowskiEngine/issues/354#issuecomment-880555195
+ - anaconda::libopenblas
```

Comment:
let's move this to pytorch-notebook/environment.yml

Reply:
Ok, done at 536d580f3f09e45a9a690294f437b4f6b5a3b55a

Reply:
fyi, the conda-forge pytorch will pull in the appropriate* blas, currently set mkl, see more here: https://github.com/conda-forge/pytorch-cpu-feedstock/blob/93333de760d702aee276dcafeedc890458a00417/recipe/meta.yaml#L79-L84

So this will likely not be useful and get overridden anyway. Conda-forge also has a matrix of blas options, so I would just get it from there.

* if you go upstream (pytorch), you will see in their own default builds, they often have precedence for mkl (mkl being more performative than others, though unfortunately not exactly open and doesn't work on all platforms). I would drop this line

Reply:
I could not get conda-lock to work without this explicitly in the environment (see https://github.com/pangeo-data/pangeo-docker-images/issues/312#issue-1215961431). If we can remove it, great--let's try! But that was not consistent with my experience. See also https://github.com/NVIDIA/MinkowskiEngine/issues/354.

Reply:
if it works, it works! :)

Reply:
I suggested an additional test below just in case